client/settings/payment-settings/index.js

```diff
@@ -24,16 +25,25 @@ const GeneralSettingsDescription = () => (
 				'woocommerce-gateway-stripe'
 			) }
 		</p>
+		<p>
+			<ExternalLink href="?TODO">
+				{ __( 'View Stripe docs', 'woocommerce-gateway-stripe' ) }
+			</ExternalLink>
+		</p>
+		<p>
+			<ExternalLink href="?TODO">
+				{ __( 'Get support', 'woocommerce-gateway-stripe' ) }
+			</ExternalLink>
+		</p>
```

Comment:
As per the design these links do not have underlines. Need to add `text-decoration: none;` in the styles.

Reply:
I pinged design and we will be keeping the underline here. The idea is to leverage off existing components at hand. In this case, WP component `< ExternalLink >` comes with the underline, so we will keep it simple and stay with it. 

Thread: p1631257369046300/1631208618.035300-slack-C01RTUVR81K.